### ISSUE TITLE
[Dapp experiment] Aragon DAO for Decentralized Software Chicago

### ISSUE THREAD

**Issue by pcowgill** [decentralized-software/chicago#32]
TL;DR: Playing with a new DAO for this group - send me your address if you want to test it out.

Long version:

I was thinking it would be cool to try experimenting with a DAO for this group using Aragon (just on Rinkeby testnet though, so no real money involved).

I initialized an org on there called `decentralizedsoftwarechicago` and set it up with democracy-style voting (as opposed to oligarchy/multi-sig).

I have it set for a threshold of 50% yes votes required for a vote to pass, 10% of accounts needed for a quorum, and a 4 day voting period for all votes. (That way if someone is offline for a long weekend, they still have time to vote when they get back.)

If you're interested in testing it out with me, I just would need to know what Ethereum address you prefer to use to play around with dapps on Rinkeby. If you're already on [Keybase](https://github.com/decentralized-software/chicago/issues/21), you can DM it to me on there.

If you'd prefer to use something other than Keybase to share your address, feel free to comment on this issue to let me know what tool you'd prefer to use to share it. I figure some people will prefer to keep their account/pseudonym as private as possible, whereas others might want it public for a chance at random airdrops - and in that case feel free to share it on this GitHub issue. But if you haven't carefully thought through your strategy / philosophy on this topic, I'd recommend defaulting to private. Also, note that when you switch from mainnet to Rinkeby using MetaMask, your Ethereum addresses in MetaMask are still the same ones. So it's worth considering creating a new, distinct address for testing dapps on testnets (but it's also a sensible choice not to do this - it's still early days so it's unclear what the opsec best practices are.)

Ideally Aragon would have a shareable link to let people opt in to a democracy-style DAO, but that feature doesn't seem to exist yet.

**Comment by pcowgill** [comment]
Here's the link for our DAO https://rinkeby.aragon.org/#/decentralizedsoftwarechicago.aragonid.eth

**Comment by pcowgill** [comment]
Anyone else interested in playing with our DAO?

**Comment by pcowgill** [comment]
@adisney @willmeister There's a vote open on our DAO to upgrade our contracts to the latest version.

https://rinkeby.aragon.org/#/decentralizedsoftwarechicago.aragonid.eth/0x8dfc96acfe826e2f422b813ea6f84619b63537af

**Comment by pcowgill** [comment]
Also, if anyone else wants to join, I'd be happy to create a vote to add you!
